tests/spread/core24/set-version-twice/task.yaml

```diff
@@ -4,7 +4,8 @@ prepare: |
   #shellcheck source=tests/spread/tools/snapcraft-yaml.sh
   . "$TOOLS_DIR/snapcraft-yaml.sh"
 
-restore: |  # If this next line fails, delete it and replace it with the one following it
+restore:
+  | # If this next line fails, delete it and replace it with the one following it
```

Comment:
Same here

Reply:
Now out of scope for this effort, a future enhancement will be to not ignore yaml files in test/spread